ansible/roles/common/tasks/main.yml

```diff
@@ -2,17 +2,24 @@
   - name: Test connection
     ping:
 
+  - name: Ensure /etc/profile.d/ exists
+    file:
+      path: "/etc/profile.d/"
+      owner: root
+      state: directory
+    become: yes
```

Comment:
You could probably find an example of a `when: is a linux machine` and put that here so we don't do this on macs since it doesn't work there.